src/modules/assignment/schema/assignment.schema.ts

```diff
@@ -3,19 +3,24 @@ import { Document } from 'mongoose';
 
 export type AssignmentDocument = Assignment & Document;
 
-@Schema()
+@Schema({ timestamps: true })
 export class Assignment {
   @Prop({ required: true })
-  name: string;
+  assignmentLink: string;
 
   @Prop({ required: true })
-  link: string;
+  assignmenDescription: string;
 
-  @Prop()
-  submit_by: number;
-
-  @Prop()
-  created_at: Date;
+  @Prop({ required: true })
+  createdBy: string;
```

Comment:
created by: mentor

Reply:
This is not solved

Reply:
yeah it was initially this, then you said to change it to string only because only name had to be displayed for assignment creator